lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/new-p2-identity.yaml

```diff
@@ -577,12 +581,23 @@ PYI_ESCAPE_M2B:
     type: page
     pageId: pyi-escape-m2b
   events:
+    end:
+      targetState: RETURN_TO_RP
     next:
       targetState: IDENTITY_START_PAGE
     bankAccount:
       targetState: BANK_ACCOUNT_START_PAGE
+
+PYI_ESCAPE_ABANDON_M2B:
+  response:
+    type: page
+    context: abandon
+    pageId: pyi-escape-m2b
+  events:
     end:
       targetState: RETURN_TO_RP
+    next:
+      targetState: IDENTITY_START_PAGE
```

Comment:
A thought, do we need/want to go through `RESET_IDENTITY` first here 🤔

Reply:
I guess it depends on what user behaviour we want. 

Do we want this to act as if you were starting the entire journey again? 

Or are we simply routing the user to the DCMAW page so they can submit passport details?

Reply:
I think we probably do rather than having orphaned CIC/BAV VCs hanging around
but be keen to get @Joe-Edwards-GDS's thoughts on this?

Reply:
To check, this is if you abandon the M2B journey in the NINO CRI? In this case it makes sense to me to reset, since the CIC/BAV VCs won't be useful, however it's not strictly necessary.

The main impact is whether we want to restrict the user to the same name/dob as they entered in CIC/BAV the first time round...

(The obvious parallel is the existing 'cri escape' route (i.e. for kbv thin file), where actually we don't clear old VCs, but in that case there is some value to reusing address and fraud, it's not just starting from scratch).

Reply:
Agreed, yeah this is abandoning at either BAV or NINO CRI

Reply:
Ok that makes sense, I'll switch it to RESET_IDENTITY then